docs/releases/3.0.3.rst

```diff
@@ -0,0 +1,56 @@
+3.0.3 Release Notes
+===================
+
+Channels 3.0.3 fixes a security issue in Channels 3.0.2
+
+CVE-2020-XXXXX: Potential leakage of session data using legacy ``AsgiHandler``
+------------------------------------------------------------------------------
+
+The legacy ``channels.http.AsgiHandler`` class, used for handling HTTP type
+requests in an ASGI environment prior to Django 3.0, did not correctly separate
+request scopes in Channels 3.0. In many cases this would result in a crash but,
+with correct timing responses could be sent to the wrong client, resulting in
+potential leakage of session identifiers and other sensitive data.
+
+This issue affects Channels 3.0.x before 3.0.3, and is resolved in Channels
+3.0.3.
+
+Users of ``ProtocolTypeRouter`` not explicitly specifying the handler for the
+``'http'`` key, or those explicitly using ``channels.http.AsgiHandler``, likely
+to support Django v2.2, are affected and should update immediately.
+
+Note that both not explicitly specifying the handler for the ``'http'`` key
```

Comment:
'both not' is a bit hard to read. Could remove the word 'both'?

Reply:
Adjusted.